NWEEI/NWEEI/Views/Programs/EnergyMgmt.cshtml

```diff
@@ -0,0 +1,173 @@
+﻿@{
+    ViewData["Title"] = "Energy Management Technician";
+}
+
+<!-- degree title and image -->
+<div>
+    <div class="energy-mgmt program-header">
+        <div class="program-title">Energy Management</div>
+    </div>
+</div>
+
+<!-- degree overview -->
+<div class="container content-section">
+    <h3>Energy Management Technician, AAS</h3>
```

Comment:
I like how the format between these degrees is uniform. Makes for very easy scanning of info.